src/pages/ContestPage.tsx

```diff
@@ -20,38 +17,37 @@ const Progress = styled(Text)`
 `
 
 interface ContestParams {
-  id: string
+  contestNumber: string
 }
 
 interface Props extends RouteComponentProps<ContestParams> {}
 
 const ContestPage = (props: Props) => {
-  const { id } = props.match.params
-  const { election, resetBallot, updateVote, votes } = useContext(BallotContext)
-  const { contests, bmdConfig } = election!
+  const { contestNumber } = props.match.params
+  const {
+    election,
+    updateVote,
+    votes,
+    contests: contestsOrNull,
+    resetBallot,
+  } = useContext(BallotContext)
+
+  const contests = contestsOrNull!
```

Comment:
This TypeScript project is configured to not allow the use of `null`, only `undefined`.